<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPrincipale</class>
 <widget class="QMainWindow" name="FenPrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenPrincipale</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/icone.ico</normaloff>:/icon/icone.ico</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widgetCentral">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,5" columnstretch="0,0,0,1">
      <item row="1" column="0" colspan="2">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,1,1,0,15,0,0,0,0,0">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="b_tb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>65</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Tableau de bord</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_graph">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Graphiques</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_carte">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Carte</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_console">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Console</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_table">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Historique</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b_param">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Paramètres</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="konami_4">
           <property name="text">
            <string>Google</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="konami_2">
           <property name="text">
            <string>G-Switch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="konami_3">
           <property name="text">
            <string>Trololo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="konami_1">
           <property name="text">
            <string>Nyan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="konami_close">
           <property name="text">
            <string>ALT-TAB ALT-TAB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QStackedWidget" name="stack">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="p_tabl_bord">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="container"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_debug_console">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Logger</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPlainTextEdit" name="console">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Série</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QTextBrowser" name="console_serie"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Décodeur</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QTextBrowser" name="console_decodeur"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Base de données</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QTextBrowser" name="console_bdd"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Web services</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QTextBrowser" name="console_webservices"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_historique">
         <layout class="QGridLayout" name="verticalLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="4" column="0">
             <widget class="QTabWidget" name="tab_historique"/>
            </item>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="p_historique_range_selection_layout"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="actualizeTableButton">
              <property name="text">
               <string>Actualiser</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_params">
         <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0">
          <item row="4" column="3" colspan="2">
           <widget class="QLineEdit" name="dataServerLineEdit">
            <property name="placeholderText">
             <string>http://127.0.0.1/cookie/bin/add.php</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QCheckBox" name="interpol_curve">
            <property name="text">
             <string>Interpoler les valeurs de courbe</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QCheckBox" name="dataServerCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="5">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>URL du serveur de réception</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QCheckBox" name="get_infos">
            <property name="text">
             <string>Activer mise à jour des infos</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Heure de lancement</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>URL du serveur MétéWow</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="5">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="5">
           <widget class="QPushButton" name="metewowRegisterPushButton">
            <property name="text">
             <string>S'enregistrer</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QLineEdit" name="metewowMacLineEdit">
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>     Identifiant/MDP</string>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QCheckBox" name="metewowServerCheckBox">
            <property name="text">
             <string/>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QLineEdit" name="metewowServerLineEdit">
            <property name="toolTip">
             <string/>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="placeholderText">
             <string>http://mtw.lortex.org/api/</string>
            </property>
           </widget>
          </item>
          <item row="7" column="4">
           <widget class="QLineEdit" name="metewowMdpLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QTimeEdit" name="heureLancement">
            <property name="displayFormat">
             <string>HH:mm:ss</string>
            </property>
           </widget>
          </item>
          <item row="8" column="5">
           <widget class="QPushButton" name="metewowDeletePushButton">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_graphics">
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="7,0,0,0">
          <item row="2" column="0">
           <widget class="QMdiArea" name="zone_graph"/>
          </item>
          <item row="3" column="0">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="sel_capteur"/>
             </item>
             <item>
              <widget class="QComboBox" name="sel_valeur"/>
             </item>
             <item>
              <widget class="QPushButton" name="add_graph">
               <property name="text">
                <string>Ajouter le graphique</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_optimiser">
               <property name="text">
                <string>OPTIMISATION</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="p_graphics_range_selection_layout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_maps">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="c_maps"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="p_konami">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_51">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="p_konami_layout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <layout class="QHBoxLayout" name="stageLayout">
        <item>
         <widget class="MyGraphicsView" name="stageGraphicsView">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0,0,0,0)</string>
          </property>
          <property name="renderHints">
           <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="stats_octets_recus">
            <property name="text">
             <string>Octets reçus: 0 </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="stats_trames_recues">
            <property name="text">
             <string>Trames reçues: 0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="stats_trames_echouees">
            <property name="text">
             <string>Trames échouées: 0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="stats_trames_corrigees">
            <property name="text">
             <string>Trames corrigées: 0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="indicator_rx">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator:checked {
	color:red;

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="barreStatus"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogger">
    <property name="title">
     <string>Logger</string>
    </property>
    <widget class="QMenu" name="menuFrequence">
     <property name="title">
      <string>Fréquence</string>
     </property>
     <addaction name="action137050"/>
     <addaction name="action137500"/>
     <addaction name="actionEntrer_le_canal_manuellement"/>
    </widget>
    <addaction name="actionOuvrir"/>
    <addaction name="menuFrequence"/>
    <addaction name="actionHaut_parleurs"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuS_rie">
    <property name="title">
     <string>Série</string>
    </property>
    <widget class="QMenu" name="menuPort">
     <property name="title">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuBaudrate">
     <property name="title">
      <string>Baudrate</string>
     </property>
     <addaction name="action600"/>
     <addaction name="action9600"/>
     <addaction name="action57600"/>
    </widget>
    <addaction name="menuPort"/>
    <addaction name="menuBaudrate"/>
    <addaction name="separator"/>
    <addaction name="actionBalayage_frequentiel"/>
   </widget>
   <widget class="QMenu" name="menu_tapes">
    <property name="title">
     <string>Étapes</string>
    </property>
    <addaction name="actionPasser_l_tape_suivante"/>
    <addaction name="actionPasser_l_tape_pr_c_dente"/>
    <addaction name="separator"/>
    <addaction name="actionRemettre_z_ro"/>
   </widget>
   <addaction name="menuLogger"/>
   <addaction name="menuS_rie"/>
   <addaction name="menu_tapes"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="action137500">
   <property name="text">
    <string>137.500 MHz</string>
   </property>
  </action>
  <action name="action137050">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>137.050 MHz</string>
   </property>
  </action>
  <action name="actionHaut_parleurs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Haut-parleurs</string>
   </property>
  </action>
  <action name="actionBalayage_frequentiel">
   <property name="text">
    <string>Balayage fréquentiel</string>
   </property>
  </action>
  <action name="actionTodo_bosser">
   <property name="text">
    <string>Todo: bosser</string>
   </property>
  </action>
  <action name="action600">
   <property name="text">
    <string>600</string>
   </property>
  </action>
  <action name="action9600">
   <property name="text">
    <string>9600</string>
   </property>
  </action>
  <action name="action57600">
   <property name="text">
    <string>57600</string>
   </property>
  </action>
  <action name="actionPasser_l_tape_suivante">
   <property name="text">
    <string>Passer à l'étape suivante</string>
   </property>
  </action>
  <action name="actionRemettre_z_ro">
   <property name="text">
    <string>Remettre à zéro</string>
   </property>
  </action>
  <action name="actionPasser_l_tape_pr_c_dente">
   <property name="text">
    <string>Passer à l'étape précédente</string>
   </property>
  </action>
  <action name="actionEntrer_le_canal_manuellement">
   <property name="text">
    <string>Canal manuel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>UI/mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
